jobs/e2e-test-cypress.groovy

```diff
@@ -18,14 +17,13 @@ import org.khanacademy.Setup;
 //import vars.withTimeout
 //import vars.withSecrets
 
-public class TestFailed extends Exception {}
 
 new Setup(steps
 
 ).addStringParam(
-   "GIT_REVISION",
-   """The name of a webapp branch to use when building.""",
-   "master"
+   "CYPRESS_GIT_REVISION",
+   """The name of a cypress branch to use when building.""",
+   "feature/cypress"
```

Comment:
Oh interesting.  Before you were checking out a separate repo but now you're not?  And `feature/cypress` is a branch on webapp?

Reply:
btw, it is working well..

Reply:
Yes, in that case what you have is exactly right.